### DISCUSSION TITLE
Property grid category button

### DISCUSSION THREAD

**Discussion by ghost** [wxFormBuilder/wxFormBuilder#776] [Ideas]
I am creating a property grid and it's going pretty well.  I looked for a way to add categories but there was no button for that.  I'd like to suggest adding a button for that.

Thanks

**Comment by sodevel** [comment]
Categories are just regular `propGridItem`'s with the `type` `Category`, so all you need to do is to select that type. However, the current wxFB datamodel does not allow to nest categories.
